Provider Provisioner/scripts/download-spctl/index.ts

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable no-console */
+import fs from 'fs';
+import path from 'path';
+import colors from 'colors';
+import { downloadSpctl } from './helpers/download-spctl';
+
+async function main(): Promise<void> {
+  const version = '0.8.8';
```

Comment:
It is needed to get last version dynamically. We will forget to update this here on every spctl version up

Reply:
implemented latest release downloading